### PR TITLE
TLWP-1352 - Scripts to delete several user's pipeline opportunities

### PR DESCRIPTION
**Pull request by mikeymonster**

### CODE REVIEW

File: Ad-Hoc SQL/tlwp-1352 Delete Pipeline Opportunities.sql [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+			LEFT JOIN ProviderVenue AS PV on r.ProviderVenueId = pv.Id
+			LEFT JOIN Provider AS P on pv.ProviderId = p.Id
+			INNER JOIN @users u on o.CreatedBy = u.username
+		WHERE
+			oi.IsSaved = 1 
+			AND oi.IsCompleted = 0
+			AND oi.IsDeleted = 0
+			order by opportunityitemid
+
+--select * from OpportunityItem
```
Comment (byronbayer) [inline Ad-Hoc SQL/tlwp-1352 Delete Pipeline Opportunities.sql:63]: Should these be in here or removed?

File: Ad-Hoc SQL/tlwp-1352 Delete Pipeline Opportunities.sql [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+			LEFT JOIN Provider AS P on pv.ProviderId = p.Id
+			INNER JOIN @users u on o.CreatedBy = u.username
+		WHERE
+			oi.IsSaved = 1 
+	AND oi.IsCompleted = 0
+	AND oi.IsDeleted = 0
+			--oi.IsSaved <> 1 
+			--or oi.IsCompleted <> 0
+			--or oi.IsDeleted <> 0
+-- opportunityId = 975
```
Comment (byronbayer) [inline Ad-Hoc SQL/tlwp-1352 Delete Pipeline Opportunities.sql:136]: Should this be removed?

File: Ad-Hoc SQL/tlwp-1352 Delete Pipeline Opportunities.sql [hunk 3]
```diff
@@ -0,0 +1,139 @@
+--TLWP-1352 - Delete pipeline opportunities for specific users
```
Comment (byronbayer) [inline Ad-Hoc SQL/tlwp-1352 Delete Pipeline Opportunities.sql:1]: Is this file required?

Comment (mikeymonster) [reply Ad-Hoc SQL/tlwp-1352 Delete Pipeline Opportunities.sql]: File removed - everything is done in the post-deployment script now.

File: Ad-Hoc SQL/tlwp-1352 Delete Pipeline Opportunities.sql (hunk 2 above)
Comment (mikeymonster) [reply Ad-Hoc SQL/tlwp-1352 Delete Pipeline Opportunities.sql]: N/A - file has been removed

File: Ad-Hoc SQL/tlwp-1352 Delete Pipeline Opportunities.sql (hunk 1 above)
Comment (mikeymonster) [reply Ad-Hoc SQL/tlwp-1352 Delete Pipeline Opportunities.sql]: N/A - file has been removed

**Review by byronbayer — APPROVED** [review]
